docs/kubernetes/backup.md

```diff
@@ -0,0 +1,174 @@
+# Filestore Backups User Guide (Beta)
+
```

Comment:
Add that Filestore Backups rely on the VolumeSnapshot feature----even though you mention it below there may be confusion on Backups vs Snapshots.

Reply:
Done